.buildbot.sh

```diff
@@ -4,12 +4,18 @@
 
 set -e
 
+case ${CI_TRACER_KIND} in
+    "sw" | "hw" ) true;;
```

Comment:
I'm not a bash expert, but this syntax looks odd. Are there really two closing brackets `)` but no opening ones?

Reply:
It's the way it is...
https://www.shellscript.sh/case.html

Reply:
Urgh!